assets/js/src/form_editor/components/history_redo.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { redo as redoIcon } from '@wordpress/icons';
+
+function HistoryRedo(props) {
+  const hasRedo = useSelect(
+    (select) => select('mailpoet-form-editor').hasEditorRedo(),
+    []
+  );
+  const { historyMove } = useDispatch('mailpoet-form-editor');
+  const redoAction = () => {
+    historyMove('redo');
```

Comment:
You can convince me otherwise, but I think it is nicer to have two different functions for undo and redo rather than using a string argument. We wouldn't have to check the value of the argument and it reads a bit better I think.

Reply:
The functions would be almost the same instead of two lines. I don't guess that it's the right way.
Maybe something like a private function and two public functions?

Reply:
Yeah, one private with two public functions sound like a great plan.